.ci.sh

```diff
@@ -7,12 +7,14 @@ apt-get install -y apt-transport-https
 
 # Setup PCP
 wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=pcp | apt-key add -
-echo "deb https://dl.bintray.com/pcp/stretch stretch main" >> /etc/apt/sources.list
+#echo "deb https://dl.bintray.com/pcp/stretch stretch main" >> /etc/apt/sources.list
```

Comment:
I guess the idea here may have been to test Parfait with the latest PCP upstream release.  With this change, we'll be relying on the PCP shipped with whichever (stable) version of Debian is used here.  I wonder if instead we should keep the bintray use (latest PCP upstream) and make this line match the platform used in travis.yml (now Ubuntu "xenial")?

It looks like in one place we are (or were before this commit) using a Debian release, and in another we're using a (different) Ubuntu release - perhaps that was contributing to the original problem here.

Reply:
ok I think I managed to get `bionic` to work, so will go with that - this gives us a better integration test across the products, so I like it this way too.

I just wanted it to work..